Move article back to Data Forwarding section (#3644)
SumoLogic/sumologic-documentation: docs/manage/data-forwarding/index.md

      <p>View, edit, delete, activate, and deactivate data forwarding destinations.</p>
    </div>
  </div>
  <div className="box smallbox card">
    <div className="container">
      <a href="/docs/manage/data-forwarding/view-list-data-forwarding/">
        <img src={useBaseUrl('img/icons/operations/send-data.png')} alt="Thumbnail icon" width="45" />
        <h4>View Information About Data Forwarding</h4>
      </a>

>       <p>Learn how to view a list of data forwarding configured for your organization, and to view the basic info and details of data forwarding.</p>

Reviewer: ```suggestion
      <p>Learn how to view data forwarding configuration information for your organization.</p>
```
